From 393a65de77f9fa41d9a599a132f9fb9e08e731be Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 9 Feb 2018 15:36:07 +0800
Subject: [PATCH] ARM: rockchip_defconfig: remove USB_OTG_WAKELOCK

cc756e682c1a ("ANDROID: android-base.cfg: remove USB_OTG_WAKELOCK")

CONFIG_USB_OTG_WAKELOCK is currently somewhat outdated
and as such is not applicable to all Android devices. Until
it is brought up to date, remove it from the base Android
kernel configuration.

Change-Id: I4d6a2ba78ddc9210bc949ee2ecc5182a4814715d
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index 226eef102434..78c5bf63feb3 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -562,7 +562,6 @@ CONFIG_USB_SERIAL=y
 CONFIG_USB_SERIAL_GENERIC=y
 CONFIG_USB_SERIAL_OPTION=y
 CONFIG_USB_TRANCEVIBRATOR=y
-CONFIG_USB_OTG_WAKELOCK=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_GADGET_DEBUG_FILES=y
 CONFIG_USB_GADGET_VBUS_DRAW=500
-- 
2.35.3

